examples/vue/src/pages/ui/components/authenticator/custom-slots/index.vue

```diff
@@ -0,0 +1,92 @@
+<script setup lang="ts">
+import Amplify from 'aws-amplify';
+import {
+  Authenticator,
+  // Vue Composable to get access to validation errors
+  useAuthenticator,
+} from '@aws-amplify/ui-vue';
+import '@aws-amplify/ui-vue/styles.css';
+import { toRefs } from 'vue';
+
+import awsExports from '@environments/auth-with-federated/src/aws-exports';
+Amplify.configure(awsExports);
+
+const { toResetPassword, toSignIn } = toRefs(useAuthenticator());
+</script>
+
+<template>
+  <authenticator>
+    <template v-slot:header>
+      <div style="padding: var(--amplify-space-large); text-align: center">
+        <img
+          class="amplify-image"
```

Comment:
I used this to give it the same look-and-feel. Since this is just for the examples, should I remove these styles so that the docs are cleaner @ErikCH ?

Reply:
That's fine with me, we can keep it.